tests/boardfarm_plugins/boardfarm_sample_plugin/tests/initial_ap_config.py

```diff
@@ -0,0 +1,42 @@
+
+import boardfarm
+
+from boardfarm.tests import bft_base_test
+
+
+class InitialApConfig(bft_base_test.BftBaseTest):
+    '''
+    PrplMesh sample test case, no actual testing, it just passes
+    '''
+
+    def runTest(self):
+        '''
+        Main test logic must be implemented here.
+        '''
+        print('{}: Running...'.format(self.__class__.__name__))
+
+    def skipTest(self, msg: str = ""):
+        '''
+        Method invoked if DUT's consoles don't respond,
+        generally considering device as 'dead'(in normal workflow).
+        Also, it can be used to provide info about possible failure reasons.
+        '''
+        print('{}: Skipping...'.format(self.__class__.__name__))
+        raise boardfarm.exceptions.SkipTest(msg)
```

Comment:
Do we need to implement this method or is it optional?

If it is optional, I would just remove it. It's mentioned in the README, that's enough.

Reply:
> Do we need to implement this method or is it optional?
> 
> If it is optional, I would just remove it. It's mentioned in the README, that's enough.

All four methods mentioned in the README are mandatory.